resources/lang/de/injuries.php

```diff
@@ -11,91 +11,108 @@
-    'Abdominal Strain'          => 'Abdominal Strain',
-    'Groin/Pelvis Injury'       => 'Groin/Pelvis Injury',
-    'Toe Injury'                => 'Toe Injury',
-    'Calf Muscle Strain'        => 'Calf Muscle Strain',
-    'Inguinal Hernia'           => 'Inguinal Hernia',
-    'Concussion'                => 'Concussion',
-    'Virus'                     => 'Virus',
-    'Head Injury'               => 'Head Injury',
-    'ACL Knee Ligament Injury'  => 'ACL Knee Ligament Injury',
-    'Shoulder Injury'           => 'Shoulder Injury',
-    'Chest Injury'              => 'Chest Injury',
-    'Rib Injury'                => 'Rib Injury',
-    'Illness'                   => 'Illness',
-    'Tonsillitis'               => 'Tonsillitis',
-    'Cartilage Knee Injury'     => 'Cartilage Knee Injury',
-    'Neck Injury'               => 'Neck Injury',
-    'Dislocated Shoulder'       => 'Dislocated Shoulder',
-    'Broken Leg'                => 'Broken Leg',
-    'Gluteal Strain'            => 'Gluteal Strain',
-    'Achilles tendonitis'       => 'Achilles tendonitis',
-    'Metatarsal Fracture'       => 'Metatarsal Fracture',
-    'Heel Injury'               => 'Heel Injury',
-    'Gashed Leg'                => 'Gashed Leg',
-    'Broken Hand'               => 'Broken Hand',
-    'Appendicitis'              => 'Appendicitis',
-    'Broken Arm'                => 'Broken Arm',
-    'Eye Injury'                => 'Eye Injury',
-    'Hip Labrum Tear'           => 'Hip Labrum Tear',
-    'Patella Fracture'          => 'Patella Fracture',
-    'LCL Knee Ligament Injury'  => 'LCL Knee Ligament Injury',
-    'Viral Meningitis'          => 'Viral Meningitis',
-    'Medical Condition'         => 'Medical Condition',
-    'Wisdom Teeth'              => 'Wisdom Teeth',
-    'Chin Injury'               => 'Chin Injury',
-    'Fractured Cheekbone'       => 'Fractured Cheekbone',
-    'Kidney Injury'             => 'Kidney Injury',
-    'Broken Toe'                => 'Broken Toe',
-    'Fibula fracture'           => 'Fibula fracture',
-    'Twisted Knee'              => 'Twisted Knee',
-    'Broken Nose'               => 'Broken Nose',
-    'Broken Collarbone'         => 'Broken Collarbone',
-    'Bowel Disease'             => 'Bowel Disease',
-    'Sciatica'                  => 'Sciatica',
-    'Testicular Cancer'         => 'Testicular Cancer',
-    'Broken Finger'             => 'Broken Finger',
-    'Rib Fracture'              => 'Rib Fracture',
-    'Ankle Fracture'            => 'Ankle Fracture',
-    'Broken Shinbone'           => 'Broken Shinbone',
-    'Broken Wrist'              => 'Broken Wrist',
-    'Hand Injury'               => 'Hand Injury',
-    'Ear Infection'             => 'Ear Infection',
+    'Unknown'                   => 'Unbekannt',
+    'Ankle/Foot Injury'         => 'Knöchelverletzung',
+    'Facial Injury'             => 'Gesichtsverletzung',
+    /*
+     * Note: I'm not sure, which tendon you mean, as the word is ambiguos.
```

Comment:
We only have one injury with tendon in it (Achilles tendon rupture) and this is translated correctly. I got this list with injuries from the API-supplier so we don't need to include kreuzbandriss. Kreuzbandriss would be torn ACL or torn PCL.

Reply:
For now this is okay. You can remove the note